common/persistence/visibilitySamplingClient.go

```diff
@@ -25,7 +25,8 @@ import (
 	"runtime"
 	"sync"
 
-	"github.com/uber/cadence/common/clock"
```

Comment:
Looks like this internal `cadence/common/clock` package is used in many places. I think it's good to have such internal wrapper. If `github.com/benbjohnson/clock` provides better functionality/testing we should switch the underlying implementation of `cadence/common/clock` package so all callers benefit from it. Using a custom clock vendor in one place and leaving others as is can be more confusing than it helps. 
WDYT about sticking with our wrapper and improving it?

Reply:
Makes sense. Let me try it out.